rules/windows/process_creation/proc_creation_win_hktl_wce.yml

```diff
@@ -22,9 +22,13 @@ detection:
     selection_2:
         CommandLine|endswith: '.exe -S'
         ParentImage|endswith: '\services.exe'
```

Comment:
This whole selection is risky. The rule cannot be critical
I suggest we re-write the rule and look for better indicators.

@swachchhanda000 remove the changes from this PR so that we can merge it and then update it in a future PR